analytics/upress_reporting/models/pollers/poller.py

```diff
@@ -51,15 +51,17 @@ def pull_interaction_events_from_logs(self, log_path, bucket_name, regex) -> lis
         events = []
         today = pandas.Timestamp.today()
 
-        aggregate_logs.aggregate_logs_in_period(self.date_range, bucket_name, 
+        aggregate_logs.aggregate_logs_in_period(self.date_range, bucket_name,
                                                 log_path, regex, self.referrer_url)
-    
+
         for date in self.date_range:
             if date > today:
-                print("No logs exist past today's date: ", today.strftime("%b %d, %Y"))
+                print("No logs exist past today's date: ",
```

Comment:
same as above

Reply:
I did a little light formatting with autopep8 to some of the base files (polller, counter_5_report) for spacing purposes since all the lines were getting very long, lmk what you think!